tifeatures/factory.py

```diff
@@ -521,6 +524,13 @@ async def items(
                 description="Starts the response at an offset.",
             ),
             output_type: Optional[ResponseType] = Depends(OutputType),
+            bbox_only: Optional[bool] = Query(
+                None,
+                description="Only return the bounding box of the feature.",
+            ),
+            simplify: Optional[float] = Query(
+                None, description="Simplify the output geometry."
+            ),
```

Comment:
`simplify` sounds more like a `bool` to me. Is the idea to change the precision of the geojson?

Reply:
It does two things, first it runs [ST_Simplify](https://postgis.net/docs/ST_Simplify.html) that runs a douglas-peuker simplification with the passed in float tolerance to reduce points, then it runs [ST_SnapToGrid](https://postgis.net/docs/ST_SnapToGrid.html) using the same tolerance that drops the precision while dropping out any coincident points that were caused by the drop in precision.

Reply:
👍 let's update the `description` to reflect that 😄